plugins/Delay/DelayControlsDialog.cpp

```diff
@@ -49,35 +49,35 @@ DelayControlsDialog::DelayControlsDialog( DelayControls *controls ) :
 	sampleDelayKnob->setVolumeKnob( false );
 	sampleDelayKnob->setModel( &controls->m_delayTimeModel );
 	sampleDelayKnob->setLabel( tr( "DELAY" ) );
-	sampleDelayKnob->setHintText( tr( "Delay time" ) + " ", " s" );
+	sampleDelayKnob->setHintText(tr("Delay time: "), " s");
```

Comment:
In some places, you use ": " with a space after the colon, while other places you use ":" without the space. This should probably be made consistent.

Reply:
Consistency was the primary goal of this pr, so it might truly be a good idea to agree upon a scheme on where exactly the spaces should be inserted as currently it varies by plugin. For now I only esured it displays right in the ui at least.